decoder/vaapidecoder_h264_dpb.cpp

```diff
@@ -55,7 +55,7 @@ static uint32_t roundLog2(uint32_t value)
 {
     uint32_t ret = 0;
     uint32_t valueSquare = value * value;
-    while ((1 << (ret + 1)) <= valueSquare)
+    while ((uint32_t)(1 << (ret + 1)) <= valueSquare)
```

Comment:
yes, because there is an compiling error. 
I think that the const number "1" shuld be the default type int, so the type of "1 << (ret + 1) " is int.